Gopkg.lock

```diff
@@ -592,6 +592,18 @@
   pruneopts = "NUT"
   revision = "2d89985ca24a8307e4b5db2574a62d3e6c2444b4"
 
+[[projects]]
+  branch = "master"
```

Comment:
Thinking we probably don't want to express a dep on `master`.

Reply:
That's probably right.